### ISSUE TITLE
Warning: Deprecated API features detected! These feature(s) are not compatible with SQLAlchemy 2.0. 

### ISSUE THREAD

**Issue by ne0YT** [myOmikron/mailcow-ldap-sync#22]
/opt/mailcow-ldap-sync/mailcow_ldap_sync/./main.py:361: MovedIn20Warning: Deprecated API features detected! These feature(s) are not compatible with SQLAlchemy 2.0. To prevent incompatible upgrades prior to updating applications, ensure requirements files are pinned to "sqlalchemy<2.0". Set environment variable SQLALCHEMY_WARN_20=1 to show all deprecation warnings.  Set environment variable SQLALCHEMY_SILENCE_UBER_WARNING=1 to silence this message. (Background on SQLAlchemy 2.0 at: https://sqlalche.me/e/b8d9)
  Base = declarative_base()

**Comment by ne0YT** [comment]
the tool still works, but gives this error.

**Comment by myOmikron** [comment]
Yeah, I have encountered this issue before when SQLAlchemy hasn't even released there 2.0 version. 

I'll check if I want to upgrade to 2.0 or pin the version to <2.0
